recipes/decli/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The license file already exists in the source so you can delete it.

Reply:
@BastianZim Thank you for reviewing! I deleting the license file and re-pushed. The conda-forge-linter is now throwing the following message: license_file entry is missing, but is required. Is this ok?

Reply:
No, you'll still need to specify the path to the license. But since the license is already included in the source distribution the path needs to point there. In your case, you can just put `license_file: LICENSE` back but not include the actual file. Conda-build will then use the one provided by the package.

Reply:
@BastianZim Copy that. I added the line back in.

Reply:
Looks good! Let me ping the relevant team for a review.